tests/PHPUnit/Integration/SegmentTest.php

```diff
@@ -271,7 +271,7 @@ public function test_getSelectQuery_whenJoinActionOnConversion()
                     *
                 FROM
                     " . Common::prefixTable('log_conversion') . " AS log_conversion
-                    LEFT JOIN " . Common::prefixTable('log_link_visit_action') . " AS log_link_visit_action ON log_conversion.idvisit = log_link_visit_action.idvisit
+                    LEFT JOIN " . Common::prefixTable('log_link_visit_action') . " AS log_link_visit_action ON log_link_visit_action.idvisit = log_conversion.idvisit
```

Comment:
The order should not be important and it was random before. Now it is at least always consistent.